src/FormStep.js

```diff
@@ -124,6 +124,7 @@ const FormStep = ({ form, submission, onStepSubmitted }) => {
           <form onSubmit={onReactSubmit}>
             <FormIOWrapper
               ref={formRef}
+              url="http://localhost:3000/bla"
```

Comment:
`url` is meant to point to a hosted form configuration on that particular and this conflicts with the `form` prop.